data/src/main/java/me/androidbox/data/remote/model/AttendeeModel.kt

```diff
@@ -0,0 +1,10 @@
+package me.androidbox.data.remote.model
+
+data class AttendeeModel(
+    val email: String,
+    val fullName: String,
+    val userId: String,
+    val eventId: String,
+    val isGoing: Boolean,
+    val remindAt: Long
+)
```

Comment:
If you follow the presentation/domain/data structure, I'd recommend to reserve model as a name for the domain layer. This model you have here represents a data model which is parsed from JSON in this case. My personal recommendation is to name data models from an API ending with Dto, so for example `AttendeeDto` (data transfer object), data models from a local DB Entity, so for example `AttendeeEntity` and models in your domain layer without and suffix, so just `Attendee`.
This is not a strict rule, but avoids any confusion with the names.

Same ofc for all other models in this PR

Reply:
@philipplackner 
Just quick question. So we should separate our data models for roomdb and api?
for example:
```
data layer (AttendeeDto, AttendeeEntity) -> for the API and DB
domain layer (AttendeeModel)
presentation (Atteendee)
```

Reply:
Exactly
Tho the one from Presentation, is completely optional. Usually on Presentation you'll also use the one from Domain